app/_src/gateway/support/index.md

```diff
@@ -73,6 +70,7 @@ These versions have reached the end of full support.
 
 | Version  | Released Date | End of Full Support | End of Sunset Support |
 |:--------:|:-------------:|:-------------------:|:---------------------:|
+|  3.0.x.x |  2022-09-09   |     2023-09-09      |      2024-09-09       |
```

Comment:
I'm not sure if the End of full support and end of sunset support dates are correct. Does anyone know where to find those?

Reply:
You've got them both right 😄 

End of full support would be the day and month of the release + 1 year.

End of sunset support is the same for all 3.x series except the LTS. The very first release in the series sets that date, which is end of full support + one year.